server/src/com/cloud/api/ApiResponseHelper.java

```diff
@@ -735,13 +740,28 @@ public VlanIpRangeResponse createVlanIpRangeResponse(Class<? extends VlanIpRange
                     vlanResponse.setPhysicalNetworkId(pnw.getUuid());
                 }
             }
+            vlanResponse.setForSystemVms(getIsForSystemVms(vlan.getId()));
             vlanResponse.setObjectName("vlan");
             return vlanResponse;
         } catch (InstantiationException | IllegalAccessException e) {
             throw new CloudRuntimeException("Failed to create Vlan IP Range response", e);
         }
     }
 
+    /**
+     * Return true if vlan IP range is dedicated for system vms (SSVM and CPVM), false if not
+     * @param vlanId vlan id
+     * @return true if VLAN IP range is dedicated to system vms
+     */
+    private boolean getIsForSystemVms(long vlanId){
```

Comment:
@nvazquez, I would suggest using the name as only isForSystemVms. Thanks!

Reply:
Done, thanks